tutorials/3d/fps_tutorial/part_six.rst

```diff
@@ -644,26 +644,26 @@ Let's go through what this function is doing.
 
 ______
 
-First we check to see if the player has just died by checking to see if ``health`` is equal or less than ``0`` and ``is_dead`` is ``false``.
+Firstly, we check whether the player has just died by checking if ``health`` is less than or equal to ``0`` and ``is_dead`` is ``false``.
 
-If the player has just died, we disable the collision shapes for the player. We do this to make sure the player is not blocking anything with their dead body.
+If the player has just died, we disable the collision shapes for the player. We do this to make sure the player is not blocking anything with his/her dead body.
```

Comment:
"their" was perfectly fine. If you really hate this gender neutral usage, you can file an issue and we can discuss it and add the consensus to the docs writing guidelines. *Then*, you can do changes according to those guidelines.

Reply:
@akien-mga : [Argumentum ad populum](https://en.wikipedia.org/wiki/Argumentum_ad_populum)? Does "Pierre est mort.  **Leur**  cadavre est sur le sol." or "Franz ist tot.  **Ihre**  Leiche liegt am Boden." sound right to you?

Reply:
This is not the place to discuss grammar rules. This is an established use in the English speaking world, and especially in the software industry, and we have no wish to fight it off.

Reply:
And *argumentum ad populum* is the root of most grammar rules. Eventually, the usage becomes the norm, which is why 0 AD Latin mutated into the wide panel of languages found nowadays in Europe.